index.html

```diff
@@ -118,7 +118,7 @@ <h5>
   </section>
   <!-- about us -->
   <section id="sixth" style="margin-bottom: 50px;">
-    <div class="container">
+    <div class="container-fluid">
```

Comment:
why is container-fluid used here?

Reply:
To remove extra space in the right and left part in desktop as well as mobile view